todo_app/data/models/trello_objects.py

```diff
@@ -0,0 +1,11 @@
+class TrelloCard:
```

Comment:
Not against this necessarily, but it looks like this primarily exists as a proxy class, would we be better off just having the `trello_repository` convert directly to our more generic `Item` class? If that would potentially mean it's easier to replace that repo with an alternative data store later with fewer changes in the trello_service that might be a useful move

(And spoiler alert, we will change to use a DB later!)

Reply:
good point. Though, now that I have removed this, the service layer provides nothing of value, I will leave it in for now as I am have been moving flats over the last few weeks so am quite behind on time. But I imagine I will remove it soon 😅